Testing/xASL_qc_TestExploreASL.m

```diff
@@ -408,37 +415,42 @@
     ResultFile{5} = xASL_adm_GetFileList(VolumeDir,'(?i)^TissueVolume.*\.tsv$','FPList');
 
 	for iFile=1:length(ResultFile) % iterate over ROI results
-		if length(ResultFile{iFile})<1
-            ResultsTable{1+iList,1+iFile} = 'empty';
-            ResultsTable{1+iList,2+iFile} = 'empty';
-            ResultsTable{1+iList,3+iFile} = 'empty';
-		elseif iFile<5 % check the ASL parameters
-            [~, TempTable] = xASL_bids_csv2tsvReadWrite(ResultFile{iFile}{end});
-            ResultsTable{1+iList,1+iFile} = TempTable{3,end-2};
-		else % check the volumetric parameters
-			[~, TempTable] = xASL_bids_csv2tsvReadWrite(ResultFile{iFile}{end});
-			% Backward compatibility:
-			% Volumetrics used to be saved as '_(L)', but this is converted by spm_jsonread to its HEX counterpart '_0x28L0x29'
-			% Now we always save to _L to avoid this. For backward compatibility we still check the old options here
-            IndexGM = find(cellfun(@(y) ~isempty(regexpi(y,'(GM_volume_L|GM_volume_(L)|GM_volume_0x28L0x29)')), TempTable(1,:)));
-            IndexWM = find(cellfun(@(y) ~isempty(regexpi(y,'(WM_volume_L|WM_volume_(L)|WM_volume_0x28L0x29)')), TempTable(1,:)));
-            IndexCSF = find(cellfun(@(y) ~isempty(regexpi(y,'(CSF_volume_L|CSF_volume_(L)|CSF_volume_0x28L0x29)')), TempTable(1,:)));
-			if ~isempty(IndexGM)
-				ResultsTable{1+iList,1+iFile} = TempTable{2, IndexGM};
-			else
-				ResultsTable{1+iList,1+iFile} = 'n/a';
-			end
-			if ~isempty(IndexWM)
-				ResultsTable{1+iList,2+iFile} = TempTable{2, IndexWM};
-			else
-				ResultsTable{1+iList,2+iFile} = 'n/a';
-			end
-			if ~isempty(IndexCSF)
-				ResultsTable{1+iList,3+iFile} = TempTable{2, IndexCSF};
-			else
-				ResultsTable{1+iList,2+iFile} = 'n/a';
-			end
-		end
+        % Make sure one individual file does not crash the table generation
```

Comment:
Totally get what you mean. The thing right here is that we run the testing for all 10 cases and after processing all of them you don't even get 50% of the results, you get nothing, because one single TSV file was damaged.